lib/mobility-core/src/Kernel/External/Verification/Interface/Types.hs

```diff
@@ -37,18 +39,27 @@ data VerifyDLAsyncReq = VerifyDLAsyncReq
 
 type VerifyDLAsyncResp = VerifyAsyncResp
 
-data VerifyRCAsyncReq = VerifyRCAsyncReq
+data VerifyRCReq = VerifyRCReq
   { rcNumber :: Text,
     driverId :: Text
   }
   deriving stock (Show, Generic)
 
-type VerifyRCAsyncResp = VerifyAsyncResp
+data VerifyRCResp = AsyncResp VerifyAsyncResp | SyncResp GovtData.GovtDataResponse
```

Comment:
Final output should be same, just saying Sync or Async response.

Reply:
done